homeassistant/components/traccar/device_tracker.py

```diff
@@ -89,6 +131,12 @@ def __init__(self, api, hass, async_see, scan_interval, custom_attributes):
         """Update info from Traccar."""
         _LOGGER.debug('Updating device data.')
         await self._api.get_device_info(self._custom_attributes)
+        self._hass.async_create_task(self.import_device_data())
```

Comment:
Why is it better here to create tasks than awaiting the coroutines directly?

Reply:
My idea is to reduce delays to a minimum by running scans in parallel. By delay i mean that the traccar events are scanned in time frames that are separated by small time gaps.
I try to be more clear: from the beginning of the scan cycle to the retrieval of data from traccar there is a small delay that may lead to miss some traccar events during that time gap. It's a very limited case, and I might even be wrong about that.

Do you have any observations/suggestions about that?

Reply:
Ok, so we let the select machinery of asyncio handle the order of the returned results? That sounds good.